src/DataGrid.tsx

```diff
@@ -232,6 +238,9 @@ function DataGrid<R, K extends keyof R, SR>({
   const [isDragging, setDragging] = useState(false);
   const [draggedOverRowIdx, setOverRowIdx] = useState<number | undefined>(undefined);
 
+  // TODO: change it to props
```

Comment:
Not sure how to change it to props as the the keys are generated internally. The format is `grandParentKey__parentKey__key__so on`

Reply:
Changed it to props for now.